### ISSUE TITLE
The standalone docker image is not working

### ISSUE THREAD

**Issue by akash07k** [squatica/selfoss-docker#2]
@squatica 
The docker-compose is working absolutely fine but I don't know why when I'm just building the image and afterwards running the container, it is not working and while opening the URL, I'm getting the 'server sent an empty response' error.
Do you have any idea that why it may happen?
I faced this same issue with my own selfoss docker images too. don't know what can cause it.

**Comment by squatica** [comment]
There should be some more detailed error message either in `docker logs` or in /var/www/html/data/logs/default.log inside the container. At the very least apache should get a hit in access log, possibly followed by an error. If you can't see even the line in access log, then it's probably a misconfiguration in the ports - are you running the container with `-p` and are you accessing the same port in your browser? Docker-compose does that for you, but if you are trying to run it alone just with docker, you'll have to bind the ports and mount volumes yourself.

**Comment by akash07k** [comment]
No, I'm currently running it via docker compose but have tried running
via image and mapping it to the correct port I.E: docker run -p
8390:80
but the same issue.
I'll check the logs too.
